backend/canvas_app_explorer/lti1p3.py

```diff
@@ -113,6 +130,14 @@ def create_user_in_django(request, message_launch):
     roles = launch_data['https://purl.imsglobal.org/spec/lti/claim/roles']
     username = custom_params['user_username']
     course_id = custom_params['canvas_course_id']
+    course_roles = custom_params['canvas_course_roles'].split(',')
+
+    staff_course_roles = [course_role for course_role in course_roles if course_role in STAFF_COURSE_ROLES]
+    user_is_course_staff = len(staff_course_roles) > 0
+    if not user_is_course_staff:
+        logger.info(f'User {username} does not have a staff role.')
+        raise PermissionDenied()
```

Comment:
Still thinking about the optimal log and error messaging and implementation here.

Reply:
maybe the log level should be WARN or ERROR?

Reply:
Changed, see https://github.com/tl-its-umich-edu/canvas-app-explorer/pull/247/commits/76a6c58050af25aba64d2af7ecd4b47a92844902